data/weather-symbol-modes/broad_categories.json

```diff
@@ -0,0 +1,4 @@
+{
+    "wet": [9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30],
+    "dry": [0, 1, 2, 3, 4, 5, 6, 7, 8]
```

Comment:
I don't think we need the nighttime symbols in these lists, as the night -> day symbol unification is done first in the summary symbol calculation.

Reply:
I've removed the night-time codes.